onepane/manifest.json

```diff
@@ -0,0 +1,77 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "68e9daf8-3439-49a0-8609-cdb91d72a036",
+  "app_id": "onepane",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Onepane is an GenAI tool that helps you faster incident resolution with automated Root Cause Analysis.",
+    "title": "Onepane",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Datadog Deploying",
+        "image_url": "images/deploying.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "List Host Resources",
+        "image_url": "images/host_resources.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Datadog Integration",
+        "image_url": "images/integration.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Datadog Resource Mapping",
+        "image_url": "images/onepane_mapping.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::AI/ML",
+      "Category::Cloud",
+      "Category::Automation",
+      "Category::AWS",
+      "Category::Azure",
+      "Category::Event Management",
+      "Category::Incidents",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```

Comment:
This is missing required fields. Please make sure to include all required fields such as: 
Submitted Data Type:: (sending data in)
Queried Data Type:: (pulling data out) *If applicable
Offering:: 

See here https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags